marimo/_plugins/stateless/status/_progress.py

```diff
@@ -379,3 +382,31 @@ def _finish(self) -> None:
         if self.remove_on_exit:
             self.progress.clear()
         self.progress.close()
+
+
+@mddoc
+def toast(
+    title: str,
+    description: str = "",
+    variant: Optional[Literal["danger"]] = None,
```

Comment:
to keep consistent with `mo.callout` - could we do `kind` instead of variant
```
kind: Optional[Literal["danger"]] = None,
```

and since Alert is internal, we can still do `Alert(variant=kind)`